src/modules/teamServices/components/singleServicePage/index.js

```diff
@@ -7,14 +7,13 @@ import services from "../../store/services.json";
 import styles from "./style.module.css";
 
 export default class SingleServicePage extends Component {
-  state = {
-    service: null,
-  };
-
-  componentWillMount() {
```

Comment:
Use `componentDidMount` and mute the state on it.

Reply:
I refactored it to the recommended way, mentioned in React docs.
https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html#initializing-state

Reply:
1. The article said that using `componentWillMount` and the simple refactor for that is to move the initialization into `componentDidMount` not the constructor.

2. This article is from 2018, and you're not fetching remote data as the data of the services will be static.

Reply:
The simplest refactor for this type of component is to move state initialization to the **constructor** or to a property initializer

Reply:
I talk about the section u attached in your comment.

Reply:
I already quoted my previous comment from the mentioned section,
anyways no problem, it was anew info for me